src/main/java/Akshan.java

```diff
@@ -0,0 +1,179 @@
+            ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░\s
+            ░▒▓████████▓▒░▒▓███████▓▒░ ░▒▓██████▓▒░░▒▓████████▓▒░▒▓████████▓▒░▒▓█▓▒░░▒▓█▓▒░\s
+            ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░      ░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░\s
+            ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░      ░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░\s
+            ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓███████▓▒░░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░\s
+            """;
+    private static final Map<CommandType, Function<String, String[]>> COMMAND_PARSERS = new HashMap<>();
+
+    static {
+        COMMAND_PARSERS.put(CommandType.TODO, Akshan::parseTodo);
+        COMMAND_PARSERS.put(CommandType.DEADLINE, Akshan::parseDeadline);
+        COMMAND_PARSERS.put(CommandType.EVENT, Akshan::parseEvent);
+    }
+
+
+    /**
+     * Prints a horizontal line.
+     */
+    private static void printLine(){
+        System.out.println("_______________________________________________________________");
+    }
+
+    /**
+     * Main method to run the Akshan bot.
+     */
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        TaskList taskList = new TaskList();
+
+        init();
+        String line = input.nextLine();
+
+        while (!line.equals(CommandType.BYE.getCommand())) {
+            processCommand(line, taskList);
+            printLine();
+            line = input.nextLine();
+        }
+        bye();
+    }
+
+    /**
+     * Initializes the bot, prints the logo and a welcome message.
+     */
+    private static void init(){
+        System.out.println(LOGO);
+        printLine();
+        System.out.println("Hello! I'm " + NAME);
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    /**
+     * Exit bot, prints goodbye sequence.
+     */
+    private static void bye(){
+        System.out.println( "Bye. Hope to see you again soon!");
+        printLine();
+    }
+
+    /**
+     * Processes the user's command.
+     *
+     * @param command The user's input command.
+     * @param taskList The list of tasks.
+     */
+    private static void processCommand(String command, TaskList taskList) {
+        String[] splitInput = command.split("/");
+        String commandTypeString = splitInput[0].split(" ")[0];
+        CommandType commandType = CommandType.fromString(commandTypeString);
+
+        switch (commandType) {
+        case LIST:
+            taskList.printList();
+            break;
+        case MARK:
+            processMarkUnmark(splitInput[0], taskList, true);
+            break;
+        case UNMARK:
+            processMarkUnmark(splitInput[0], taskList, false);
+            break;
+        case TODO:
+        case DEADLINE:
+        case EVENT:
+            processTask(commandType, command, taskList);
+            break;
+        default:
+            System.out.println("Uh oh, no command found in: " + command);
+        }
```

Comment:
I really like the use of switch/case to keep your logic concise and readable. 😄 A suggestion would be to add some validation for the splitInput to avoid potential issues with malformatted commands.

Reply:
Thanks for the advice! Will take note on the point of adding validation!